README.md

```diff
@@ -74,32 +76,34 @@ well known in the Linux community.)
 [your text here]
 
 -------------------------------------------------------------------------------
-### If bootloader, shim loading is, GRUB2: is CVE-2020-14372, CVE-2020-25632, CVE-2020-25647, CVE-2020-27749, CVE-2020-27779, CVE-2021-20225, CVE-2021-20233, CVE-2020-10713, CVE-2020-14308, CVE-2020-14309, CVE-2020-14310, CVE-2020-14311, CVE-2020-15705, and if you are shipping the shim_lock module CVE-2021-3418
+### If shim is loading GRUB2 bootloader what exact implementation of Secureboot in GRUB2 do you have? (Either Upstream GRUB2 shim_lock verifier or Downstream RHEL/Fedora/Debian/Canonical-like implementation)
 -------------------------------------------------------------------------------
 [your text here]
 
 -------------------------------------------------------------------------------
-### What exact implementation of Secureboot in GRUB2 ( if this is your bootloader ) you have ?
-### * Upstream GRUB2 shim_lock verifier or * Downstream RHEL/Fedora/Debian/Canonical like implementation ?
--------------------------------------------------------------------------------
-[your text here]
+### If shim is loading GRUB2 bootloader and your previously released shim booted a version of grub affected by any of the following CVEs:
+* CVE-2020-14372
+* CVE-2020-25632
+* CVE-2020-25647
+* CVE-2020-27749
+* CVE-2020-27779
+* CVE-2021-20225
+* CVE-2021-20233
+* CVE-2020-10713
+* CVE-2020-14308
+* CVE-2020-14309
+* CVE-2020-14310
+* CVE-2020-14311
+* CVE-2020-15705
+* if you are shipping the shim_lock module: CVE-2021-3418
```

Comment:
Maybe put the "if" clause in parentheses to emphasize that this only affects some cases?

Reply:
You're suggesting:
```
...
* CVE-2020-14311
* CVE-2020-15705
* (if you are shipping the shim_lock module: CVE-2021-3418)
```

?

Reply:
Or even "CVE-2021-3418 (if you are shipping...)"